src/components/attendee/InputSelect/InputSelect.module.scss

```diff
@@ -28,5 +32,29 @@
   box-sizing: border-box;
   width: 20%;
   border: none;
-  border-right: 5px solid transparent;
+  border-right: border.$radius--xs solid transparent;
+}
+
+.errorIcon {
+  background-image: url(/assets/icons/input-error-icon.svg);
+  height: $-error-icon-size;
+  width: $-error-icon-size;
+  margin-left: -(space.empty(2));
+}
+
+.inputWrapper {
+  width: 100%;
+  display: flex;
+  align-items: center;
+}
+
+.invalid {
+  input {
+    @include error.input-error;
```

Comment:
[NOTE] Not sure if a single module/file named `error.scss` for a single mixin named `input-error` is the way to continue.
I guess the question is:
- Are we expecting more and different error effects or are we expecting more and different input effects?

Reply:
I can imagine that we'll have more `error` effects to style in the future, since `input` validation isn't the only thing that would be used for that